Capture/CMakeLists.txt

```
@@ -51,6 +51,9 @@ execute_process(
 set(ARCH x64)
 set(MWCAPTURE_SDK_HOME 3rdparty/MWCapture_SDK_Linux-3_3_1_1313)
 
+# capture utilities installation directory
+set(CAPTURE_INSTALL_DIR /usr/local/bin)
```

there must be some "standard" folder `cmake` assumes for installation , do not hardcode -- we might want to install to `/usr/bin` whenever doing for packaging. ... I think it should be smth like

```
```suggestion
set(CAPTURE_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/bin)
```

if that is the way to reuse `cmake` variable (haven't used it for too long, forgot)

---

Yes, it's good point, thanks, added it and additional message to console, so now it looks like:

sudo cmake --install .
-- Install configuration: ""
-- Installing ReproStim 1.8.0.225 to: /usr/local/bin
-- Installing: /usr/local/bin/reprostim-screencapture
-- Set runtime path of "/usr/local/bin/reprostim-screencapture" to ""
-- Installing: /usr/local/bin/reprostim-videocapture
-- Set runtime path of "/usr/local/bin/reprostim-videocapture" to ""